src/main/java/hudson/plugins/gradle/injection/InjectionUtil.java

```diff
@@ -0,0 +1,53 @@
+package hudson.plugins.gradle.injection;
+
+import hudson.model.labels.LabelAtom;
+import hudson.plugins.gradle.util.CollectionUtil;
+import hudson.util.FormValidation;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+public final class InjectionUtil {
+
+    private InjectionUtil() {
+    }
+
+    public static boolean isNoOk(FormValidation validation) {
+        return !isOk(validation);
+    }
+
+    public static boolean isOk(FormValidation validation) {
+        return validation.kind == FormValidation.Kind.OK;
+    }
+
+    public static boolean isAnyNotOk(FormValidation... validations) {
```

Comment:
Maybe use valid/invalid instead of ok/notOk.

Reply:
Agree. Will rename.